src/main/java/com/microsoft/jenkins/keyvault/SecretCertificateCredentials.java

```diff
@@ -73,7 +73,7 @@ public Secret getPassword() {
     @NonNull
     @Override
     public KeyStore getKeyStore() {
-        final SecretBundle secretBundle = getKeyVaultSecret();
+        final KeyVaultSecret secretBundle = getKeyVaultSecret();
```

Comment:
Since the new sdk no more has the `SecretBundle` type, change the variable name to prevent confustion.